### PR TITLE
WIP: Bottom navigation bindings

### PR DESCRIPTION
**Pull request by bernalrs**
Ready for review.

### CODE REVIEW

**Review by pahund — APPROVED** [review]

File: src/MaterialUI.re [hunk 1]
```diff
@@ -113,6 +115,74 @@ module Badge = {
     );
 };
 
+module BottomNavigationButton = {
```
Comment (InsidersByte) [inline src/MaterialUI.re]: renamed to BottomNavigationAction

File: src/MaterialUI.re [hunk 2]
```diff
@@ -113,6 +115,74 @@ module Badge = {
...
+  [@bs.module "material-ui/BottomNavigation"] external reactClass : ReasonReact.reactClass =
+    "BottomNavigationButton";
+  let make =
+      (
+        ~classes: option(string)=?,
+        ~className: option(string)=?,
+        ~icon: option([ | `String(string) | `ReactElement(ReasonReact.reactElement)])=?,
```
Comment (InsidersByte) [inline src/MaterialUI.re]: need to double check this works correctly

File: src/MaterialUI.re [hunk 3]
```diff
@@ -113,6 +115,74 @@ module Badge = {
...
+        ~label: option(string)=?,
+        ~onChange: option((ReactEventRe.Form.t => unit))=?,
+        ~onClick: option((ReactEventRe.Mouse.t => unit))=?,
+        ~selected: option(bool)=?,
+        ~showLabel: option(bool)=?,
+        ~value: option([ | `String(string) | `Int(int) | `Float(float)])=?,
```
Comment (InsidersByte) [inline src/MaterialUI.re]: this should be **'a**

File: src/MaterialUI.re [hunk 4]
```diff
@@ -113,6 +115,74 @@ module Badge = {
...
+  [@bs.module "material-ui/BottomNavigation"] external reactClass : ReasonReact.reactClass =
+    "default";
+  let make =
+      (
+        ~classes: option(Js.t({..}))=?,
+        ~className: option(string)=?,
+        ~onChange: option((ReactEventRe.Form.t => unit))=?,
+        ~showLabels: option(bool)=?,
+        ~style: option(ReactDOMRe.style)=?,
+        ~value: option([ | `Int(int) | `String(string) | `Float(float)])=?,
```
Comment (InsidersByte) [inline src/MaterialUI.re]: this should be **'a** and above **style**

File: src/MaterialUI.re [hunk 5]
```diff
@@ -113,6 +115,74 @@ module Badge = {
...
+      ) =>
+    ReasonReact.wrapJsForReason(
+      ~reactClass,
+      ~props=
+        Js.Nullable.(
+          {
+            "classes": from_opt(classes),
+            "className": from_opt(className),
+            "onChange": from_opt(onChange),
+            "showLabels": from_opt(showLabels),
```
Comment (InsidersByte) [inline src/MaterialUI.re]: this should use unwrap_bool

**Review by InsidersByte — CHANGES_REQUESTED** [review]

**Comment by InsidersByte** [comment]
this will be added in #55

**Pull request closed without merging**